e2e/utils/utils.go

```diff
@@ -82,10 +88,20 @@ func WaitForPod(clientset *kubernetes.Clientset, namespace string, name string,
 			return fmt.Errorf("pod %v is not %v phase: %v", podUpdated.Name, phase, podUpdated.Status.Phase)
 		}
 		return nil
-	}, 120*time.Second, 5*time.Second).Should(BeNil())
+	}, Duration, PollingInterval).Should(BeNil())
 	return podUpdated
 }
 
+func WaitForPodNotToBeCreated(clientset *kubernetes.Clientset, namespace string, name string) {
+	defer GinkgoRecover()
+	Consistently(func() int32 {
+		_, err := clientset.Core().Pods(namespace).Get(name)
+		statusError, ok := err.(*errors.StatusError)
+		Expect(ok).To(BeTrue())
+		return statusError.ErrStatus.Code
+	}, Duration, PollingInterval).Should(BeNumerically("==", 404))
+}
```

Comment:
This is way too long. 5 seconds should be enough to guarantee that pod is not created